common/network-common/src/main/java/org/apache/spark/network/util/JavaUtils.java

```diff
@@ -383,6 +383,7 @@ public static File createTempDir() throws IOException {
   public static File createTempDir(String root, String namePrefix) throws IOException {
     if (root == null) root = System.getProperty("java.io.tmpdir");
     if (namePrefix == null) namePrefix = "spark";
+    namePrefix = namePrefix + " with a space";
```

Comment:
I like the methodology of ensuring that our test paths contain spaces or special characters.

Should we have a `Utils.isTesting` to flag this only for test code? Are there other methods for creating temporary files in tests that should also be updated?

Reply:
In my brief search I found that they all came to this method in the end (withTempDir, etc). I'm actually undoing this change for now, because it causes too many failures and I don't have the bandwidth to fix the tests right now.
Here is a PR in my branch where I made only this change: https://github.com/databricks-david-lewis/spark-opensource/pull/2.
I am hoping to fix the problems with spark's tests, and then propose that as a PR.